### ISSUE TITLE
[openssl-windows] Avoid to install documentation and misc files 

### ISSUE THREAD

**Issue by traversaro** [microsoft/vcpkg#10661]
**Is your feature request related to a problem? Please describe.**

Whenever I install openssl-windows, I find a lot (~7000)  of html files installed in  `x64-windows\html` and `x64-windows\debug\html` . This seems to be some kind of automatically-generated API documentation, but I am not aware of any other  port that install this kind of material, and for openssl the 7000  files can quite slow down the process of compressing and extracting the installation directory. 

**Proposed solution**
I would  like to remove this files after openssl has been built.

**Describe alternatives you've considered**
An alternative would be to have  an optional feature to control the build of documentation, but i do not know if the additional complexity is worth.

**Comment by Neumann-A** [comment]
target from the makefile:
`install: install_sw install_ssldirs install_docs`

so instead of calling 
```
    vcpkg_execute_required_process(
        COMMAND nmake -f ${OPENSSL_MAKEFILE} install
        .....
```
try
```
    vcpkg_execute_required_process(
        COMMAND nmake -f ${OPENSSL_MAKEFILE} install_sw install_ssldirs
...
```

**Comment by PhoebeHui** [comment]
Thanks @traversaro @Neumann-A!

I summited a PR for the solution.

**Comment by traversaro** [comment]
> Thanks @traversaro @Neumann-A!
> 
> I summited a PR for the solution.

Great, thanks a lot!

**Comment by JackBoosY** [comment]
Hi @traversaro, thanks for reporting this issue!
This issue has been fixed in #10743, please update vcpkg and rebuild openssl.

Thanks.
